src/util/assertString.js

```diff
@@ -0,0 +1,11 @@
+export default function assertString(input) {
+  if (typeof input !== 'string') {
+    if (process.env.NODE_ENV === 'node') {
```

Comment:
Are you referring to this line?
It is needed so that depd is only included in node builds, check the `build:node` script where `NODE_ENV` is explicitly set to `node`, a babel plugin is then used to inline this value so this becomes `if ('node' === 'node') > if (true)` and then get removed by uglify, likewise the whole block is removed when `NODE_ENV=browser` (check the `build:browser` script) so we do not bundle `depd` in browser builds.